### ISSUE TITLE
Can't send messages if chat id is a negative number

### ISSUE THREAD

**Issue by lurppa** [cbrgm/telegram-github-action#32]
Action fails and prints `error: missing value for --to` if chat id contains a minus sign at the beginning.

**Comment by cbrgm** [comment]
Oh is this even possible that chat id's can be negative numbers?

**Comment by cbrgm** [comment]
Can you provide an example worklow? Have you tried to set the negative number using `'`? Example: `--to: '-123'`?

**Comment by lurppa** [comment]
Apparently all group chat id's are negative https://github.com/TelegramBots/Telegram.Bot/issues/661 .

Here's a link to the action in question: https://github.com/Linkkijkl/linkki-web/blob/main/.github/workflows/telegram_notify.yml .
```
to: ${{ secrets.TELEGRAM_CHAT_ID }}
```
I have already tried double quoting the secrets value but it got rejected by Telegram with 400 bad request, the same rejection what it replies with an unknown chat id.

I'll investigate more once I get back on my laptop. Thank you for you swift reply :)

**Comment by CubaJas** [comment]
same issue. @lurppa any update on it?

**Comment by cbrgm** [comment]
Fixed in #55. I'll publish a release in a a few minutes.

**Comment by cbrgm** [comment]
The actual issue was 
```
    - --to
    - ${{ inputs.to }}
```

in the action.yml, which needs to be in the format
```
    - --to=${{ inputs.to }}
 ```

because bash doesn't like negative numbers passed as args like this `--to "-1234..."`.
